### DISCUSSION TITLE
账号被停用

### DISCUSSION THREAD

**Discussion by limgyang** [cg3s/qa#54] [问答]
有相当长一段时间没再使用CGSSS的服务，当时离开的原因是因为速度太慢，测速只能5-20M，但我买的又是专业版。准备回来买个套餐看看现在的情况，发现原有账户被提示已被停用，改了密码也依旧提示已被停用，请设置一下启用。
ID：monkeyxxx@163.com 2001

**Comment by cg3s** [comment]
被封原因为您邀请的用户与我们产生 谩骂、不友好行为。

**Comment by limgyang** [comment]
……好吧，外国人？中国人？

**Reply by cg3s** [reply]
国人
